extensions/wikia/AdEngine/js/video/uapVideo.js

```diff
@@ -35,6 +35,14 @@ define('ext.wikia.adEngine.video.uapVideo', [
 		};
 	}
 
+	function shouldUseMegaAdUnitBuilder(type) {
+		var uapTypesMap = {
```

Comment:
Just wanted to keep things more readable instead taking care about of performance in this case ;)